archiver.go

```diff
@@ -172,6 +177,34 @@ func (deb *ArchiveDeb) RelaxedUnpack(destination string) error {
 	return deb.Unpack(destination)
 }
 
+// Helper function for packing with external tool
+func pack(command []string) error {
+	return Command{}.Run("Pack", command...)
+}
+
+func (arc *ArchiveBase) Pack(destination string) error {
+	return fmt.Errorf("Pack is not supported for '%s'", arc.file)
+}
+
+func (tar *ArchiveTar) Pack(destination string) error {
+	command := []string{"tar"}
+	command = append(command, "-c")
+
+	if compression, ok := tar.options["tarcompression"]; ok {
```

Comment:
If compression wasn't set it needs to default to gz in this case; Though it'd be nicer if one could rely on tarcmpression to always be set (or really just a filed in TarArchive) and have it default to gz.